app/models/vaccine.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+# Represents a vaccination for a Patient
+class Vaccine < ApplicationRecord
+  belongs_to :patient
+
+  VACCINE_STANDARDS = Rails.configuration.vaccine_standards.freeze
+  MAX_DOSE_NUMBER = 6
```

Comment:
Worth putting in a config?

Reply:
You mean like the vaccine information already is? We could, though this number wasn't necessarily based on maintained info from CDC but rather determined internally. Would be pretty easy to add as another config though